pallets/asset/src/lib.rs

```diff
@@ -1203,7 +1203,10 @@ impl<T: Config> Module<T> {
         // Find first byte not printable ASCII.
         let good = bytes
             .iter()
-            .position(|b| !matches!(b, 32..=126))
+            .position(|b| {
+                let deref_b = *b;
+                !deref_b.is_ascii_alphanumeric()
```

Comment:
Parentheses can be used to avoid the local variable.
```suggestion
                !(*b).is_ascii_alphanumeric()
```